### PR TITLE
change all un-asserted error printing

### PR DESCRIPTION
**Pull request by cyberliem**

### CODE REVIEW

**Comment by cyberliem** [comment]
[ReserveCore.Deposit: Core ----------> Deposit log: error: %!s(<nil>)](https://trello.com/c/Un0N5VWg/159-reservecoredeposit-core-deposit-log-error-snil)

File: core/reserve_core.go [hunk 1]
```diff
@@ -378,7 +378,7 @@ func (self ReserveCore) SetRates(
 				err = errors.New("Couldn't get mined nonce of set rate operator")
 			} else {
 				oldNonce, oldPrice, count, err = self.pendingSetrateInfo(minedNonce)
-				log.Printf("old nonce: %v, old price: %v, count: %d, err: %v", oldNonce, oldPrice, count, err)
+				log.Printf("old nonce: %v, old price: %v, count: %d, err: %v", oldNonce, oldPrice, count, common.ErrorToString(err))
```
Comment (halink0803) [inline core/reserve_core.go]: If this is ErrorToString(err), it should be %s.

File: data/fetcher/fetcher.go [hunk 2]
```diff
@@ -646,7 +646,7 @@ func (self *Fetcher) FetchStatusFromExchange(exchange Exchange, pendings []commo
 					continue
 				}
 				status, err = exchange.DepositStatus(id, txHash, currency, amount, timepoint)
-				log.Printf("Got deposit status for %v: (%s), error(%v)", activity, status, err)
+				log.Printf("Got deposit status for %v: (%s), error(%v)", activity, status, common.ErrorToString(err))
```
Comment (halink0803) [inline data/fetcher/fetcher.go]: Same here.

File: data/fetcher/fetcher.go [hunk 3]
```diff
@@ -669,7 +669,7 @@ func (self *Fetcher) FetchStatusFromExchange(exchange Exchange, pendings []commo
 					continue
 				}
 				status, tx, err = exchange.WithdrawStatus(id.EID, currency, amount, timepoint)
-				log.Printf("Got withdraw status for %v: (%s), error(%v)", activity, status, err)
+				log.Printf("Got withdraw status for %v: (%s), error(%v)", activity, status, common.ErrorToString(err))
```
Comment (halink0803) [inline data/fetcher/fetcher.go]: Here also.

File: exchange/binance/binance_endpoint.go [hunk 4]
```diff
@@ -107,7 +107,7 @@ func (self *BinanceEndpoint) GetResponse(
 		err = fmt.Errorf("Binance return with code: %d - %s", resp.StatusCode, response.Msg)
 	}
 	if err != nil || len(respBody) == 0 || rand.Int()%10 == 0 {
-		log.Printf("request to %s, got response from binance (error or throttled to 10%%): %s, err: %v", req.URL, common.TruncStr(respBody), err)
+		log.Printf("request to %s, got response from binance (error or throttled to 10%%): %s, err: %v", req.URL, common.TruncStr(respBody), common.ErrorToString(err))
```
Comment (halink0803) [inline exchange/binance/binance_endpoint.go]: Update here also.

File: exchange/bittrex_test.go [hunk 5]
```diff
@@ -41,7 +41,7 @@ func (self testBittrexInterface) CancelOrder(uuid string) (Bittcancelorder, erro
 func (self testBittrexInterface) DepositHistory(currency string) (Bittdeposithistory, error) {
 	res := Bittdeposithistory{}
 	err := json.Unmarshal([]byte(self.DepositHistoryMock), &res)
-	fmt.Printf("%v\n", err)
+	fmt.Printf("%v\n", common.ErrorToString(err))
```
Comment (halink0803) [inline exchange/bittrex_test.go]: Remember to change it to %s.

**Pull request merged**